org.gyunaev.Birdtray.appdata.xml

```diff
@@ -0,0 +1,66 @@
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.gyunaev.Birdtray</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Birdtray</name>
+  <summary>System tray new mail notification for Thunderbird</summary>
+  <description>
+    <p>
+      Birdtray is a system tray new mail notification for Thunderbird, which does not require extensions
+    </p>
+  </description>
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apa.html -->
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apas02.html -->
+  <categories>
+    <category>Calendar</category>
+    <category>Chat</category>
+    <category>Communication</category>
+    <category>ContactManagement</category>
+    <category>Email</category>
+    <category>Feed</category>
+    <category>InstantMessaging</category>
+    <category>IRCClient</category>
```

Comment:
This is an irc client?

Reply:
This is just an indicator application which monitors thunderbird accounts for new mails. So, I just copied this section from the org.mozilla.Thunderbird appdata. I don't know why thunderbird is listed as irrclient, maybe it has some extension for this. Should I change it?